electron/main.js

```diff
@@ -86,6 +86,9 @@ if (process.platform === 'win32') {
   });
 }
 
+if (shouldQuit) {
+  app.quit();
+}
```

Comment:
It seems to be working ok on my Manjaro box. What about something like: 

```
if (process.platform === 'linux' && shouldQuit) {
  app.quit();
}
```

Should that be a single `&` or double `&&` in JS?